pwa/src/components/badgeCounter/BadgeCounter.module.css

```diff
@@ -0,0 +1,18 @@
+.content {
+  display: flex;
+}
+
+.badge {
+  height: var(--web-app-font-size-lg);
+  width: var(--web-app-font-size-lg);
+  border-radius: 50%;
+  font-size: var(--web-app-font-size-xs);
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  background-color: var(--web-app-badge-counter-background-color);
+  color: var(--web-app-badge-counter-color);
+}
+.maxNumber {
+  font-size: var(--web-app-font-size-2xs);
+}
```

Comment:
I don't think `.maxNumber` is doing anything (it's not set anywhere; not should it. I think it's leftover from the previous implementation).